controllers/jobs.js

```diff
@@ -19,11 +31,48 @@ const createJob = async (req, res) => {
 }
 
 const updateJob = async(req, res) => {
-  res.send('Update job');
+  const {
+    body: { company, position },
+    user: { userId },       // nested destructuring - getting from Auth middleware
+    params: { id: jobId }   // destructure the id from the route  '/:id' and rename to 'jobId'
+  } = req;
+
+  if (company === '' || position === '') {
+    throw new BadRequestError('Company or position cannot be empty');
+  }
+
+  const job = await Job.findByIdAndUpdate(
```

Comment:
This doesn't work correctly.  findByIdAndUpdate is what the presenter says to use -- but see what I wrote in the Kangaroo discussion channel about this!  You have a security error here!